WORKSPACE

```diff
@@ -399,6 +399,13 @@ container_pull(
     repository = "kubevirt/fedora-realtime-container-disk",
 )
 
+container_pull(
+    name = "busybox",
+    registry = "index.docker.io",
```

Comment:
Dockerhub has rate limitation AFAIR, quay.io should be preferred.
Additionally, I think its preferable to use an existing image as a base.
@dhiller @brianmcarey could you please advise?

Reply:
please note we need an image with `/bin/sh`
if we have it on quay i can move there
please note it affects only upstream / release upstream